crud/base.py

```diff
@@ -8,6 +9,19 @@
 from crispy_layout_mixin import CrispyLayoutFormMixin, get_field_display
 
 
+def _form_invalid_message(msg):
+    return '%s %s' % (_('Formulário inválido.'), msg)
+
+FORM_MESSAGES = {'create': (_('Registro criado com sucesso!'),
+                            _('O registro não foi criado.')),
+                 'update': (_('Registro alterado com sucesso!'),
+                            _('Suas alterações não foram salvas.')),
+                 'delete': (_('Registro excluído com sucesso!'),
+                            _('O registro não foi excluído.'))}
+FORM_MESSAGES = {k: (a, _form_invalid_message(b))
+                 for k, (a, b) in FORM_MESSAGES.items()}
+
```

Comment:
O tipo do primeiro valor da tupla está sendo django.utils.functional.lazy.<locals>.__proxy__ ... e o segundo é uma string.. veja:
`dict: {'create': (django.utils.functional.lazy.<locals>.__proxy__ object at 0x7f4804fc8630>, 'Formulário inválido. O registro não foi criado.'), 'delete': (<django.utils.functional.lazy.<locals>.__proxy__ object at 0x7f4804f60208>, 'Formulário inválido. O registro não foi excluído.'), 'update': (<django.utils.functional.lazy.<locals>.__proxy__ object at 0x7f4818778550>, 'Formulário inválido. Suas alterações não foram salvas.')}`

não seria melhor apenas isso:
`dict: {'delete': ('Registro excluído com sucesso!', 'Formulário inválido. O registro não foi excluído.'), 'create': ('Registro criado com sucesso!', 'Formulário inválido. O registro não foi criado.'), 'update': ('Registro alterado com sucesso!', 'Formulário inválido. Suas alterações não foram salvas.')}`

isso resolve?
`FORM_MESSAGES = {k: (str(a), _form_invalid_message(b))
                 for k, (a, b) in FORM_MESSAGES.items()}`